AppleModels/update_generated.py

```diff
@@ -119,32 +117,29 @@ def validate_products(db, dbpd):
     for sysname in to_add:
       for info in db:
         if sysname in info['Specifications']['SystemReportName']:
-          print("New AppleModelCode for {}:".format(info['SystemProductName']))
+          print(f"New AppleModelCode for {info['SystemProductName']}:")
           for model in to_add[sysname]:
-            print("  - \"{}\"".format(model))
+            print(f"  - \"{model}\"")
 
 def export_db_macinfolib(db, path, year=0):
-  """
+  '''
   Export yaml database to MacInfoLib format.
   TODO: use jinja2?
-  """
+  '''
 
   with open(path, 'w') as fh:
-    print('// DO NOT EDIT! This is an autogenerated file.', file=fh)
-    print('#include "MacInfoInternal.h"', file=fh)
-    print('CONST MAC_INFO_INTERNAL_ENTRY gMacInfoModels[] = {', file=fh)
+    fh.write('// DO NOT EDIT! This is an autogenerated file.')
```

Comment:
Does this add newlines?

Reply:
Oh, I forgot to add the newline character. I'll update the code as soon as possible

Reply:
Commit 5aa2d4ec should have fixed what you wanted

Reply:
How is this change better than what we had though? Makes me feel it is a useless change that makes things less readable.

Reply:
Mmmh, you're right. I'm gonna revert to the old write statements